src/pages/regulatory/index.tsx

```diff
@@ -137,7 +137,7 @@ const Regulatory = (locale: RegulatoryProps) => {
                     <Box>
                         <Text>
                             <Localize
-                                translate_text="Deriv Investments (Europe) Limited - W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR 9033, Malta - is licensed in Malta (<0>licence no. IS/70156</0>) and authorised by the Malta Financial Services Authority under the Investments Services Act to provide investment services in the European Union."
+                                translate_text="Deriv Investments (Europe) Limited - W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR 9033, Malta - is licensed in Malta (<0>licence no. IS/70156</0>) and authorised by the Malta Financial Services Authority under the Investments Services Act to provide investment services in the European Union.It is also authorised and subject to limited regulation by the Financial Conduct Authority in the UK."
```

Comment:
Hi @aswathy-deriv just to check , what about the translation for this section? 🙏 


<img width="957" alt="Screen Shot 2022-02-10 at 10 33 00 AM" src="https://user-images.githubusercontent.com/64970259/153355139-ae611a53-5fc3-486b-98b1-4bebc41aac46.png">

Reply:
Yeah @mitra-fs  I'm checking with the compliance but I'm pretty sure it will require translation

Reply:
Not sure.We have added need translation tag.Waiting for the compliance team to respond.